modin/pandas/test/dataframe/test_default.py

```diff
@@ -358,6 +358,14 @@ def comparator1(df1, df2):
     )
 
 
+@pytest.mark.parametrize("numeric_only", [True, False, None])
```

Comment:
Can we just add this parameter to the test above?

Reply:
We can, but then the parameter matrix will increase (although this is not necessary), and I would like not to do this.